content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md

```diff
@@ -0,0 +1,62 @@
+---
+title: The XSF signs Open Letter to the European Commission
+date: 2024-01-05
+author: XMPP Standards Foundation
+categories: ["XSF Organisational", "FOSDEM"]
+---
+
+As currently many other organisations doing, the XMPP Standards Foundation (XSF) has decided to also sign the Open Letter to the European Commission initiated by [Petites Singularités](https://ps.zoethical.org/pub/lettre-publique-aux-ncp-au-sujet-de-ngi/) and translated to English by [OW2](https://www.ow2.org/view/Events/The_European_Union_must_keep_funding_free_software_open_letter)
+
+# The European Union must keep funding free software
+
+Initially published by [petites singularités](https://ps.zoethical.org/pub/lettre-publique-aux-ncp-au-sujet-de-ngi/). English translation provided by [OW2](https://www.ow2.org/view/Events/The_European_Union_must_keep_funding_free_software_open_letter). 
+
+If you want to sign the letter, please publish the letter on your website and complete the [table below](https://pad.public.cat/lettre-NCP-NGI).
```

Comment:
There is no "table below"

Reply:
Thanks, but I will ignore it as it is not from us.